### DISCUSSION TITLE
关于异常断流的问题

### DISCUSSION THREAD

**Discussion by Danny-Ken** [kkkgo/PaoPaoGateWay#92] [问和答]
使用的是paopaodns与paopaogw搭配使用，内核换了最新的1.18.3，不知道为啥，最近paopaogw更新了最新版后(用的是前置嗅探模式，改用普通模式问题同在)，换不同协议的节点都存在这个问题，看Youtube视频会出现2到5分钟后就断流的情况，需要刷新网页重新播放(网页与客户端都存在)，并且此问题仅出现使用手机或者电视盒子的时候，用电脑端网页看没问题，并且手机使用V2rayNg同样的网络、同样的节点，没有这个问题出现，换了用Openwrt的Passwall也没有这个问题，希望大佬帮忙看看什么问题

**Comment by kkkgo** [comment]
1、不更换使用原始内核试试
2、试试udp_enable=no、嗅探sniff=yes、fast_node=yes
3、检测手机是否有被分配了其他DNS地址或者IPv6地址。
4、给手机设置静态IP地址，网关填写ppgw的IP，DNS填写ppgw的DNS，测试是否会断流，如果不会，可以看看流量记录是否请求了第三方DNS。

**Reply by Danny-Ken** [reply]
测试找到问题所在了，是我的ROS的ddos规则影响了，没注意看ros的日志，另有个问题想请教下大佬，docker pull是不是不支持fakeip，不把docker.io加入dnscrypt清单中就无法拉取镜像

**Reply by kkkgo** [reply]
肯定支持的，我天天pull都没问题。
![image](https://github.com/kkkgo/PaoPaoGateWay/assets/4530897/32c1c286-b078-4384-a649-6361f592e283)
加入dnscrypt就是不走代理，那就要看看你线路是不是有问题了。
